configure

```diff
@@ -3408,6 +3435,19 @@ if test -n "$_host"; then
 			_mt32emu=no
 			_vkeybd=yes
 			;;
+		switch)
+			_backend="switch"
+			_opengl_mode=gles2
+			_build_scalers=yes
```

Comment:
`_build_scalers`, `_build_hq_scalers`, `_dynamic_modules`, `_mt32emu`, `_vorbis`, `_tremor` and `_eventrec` don't need to be set here as they are already set appropriately by default.

Reply:
fixed in 4fce992